doc/rst/language/spec/modules.rst

```diff
@@ -776,18 +781,20 @@ secondary methods are visible to any instance of the type regardless of import
 statements, see :ref:`Method_Calls`.
 
 Within an ``unqualified-list``, a visible symbol from that module may optionally
-be given a new name using the ``as`` keyword.  This new name will be usable from
-the scope of the import in place of the old name unless the old name is
-additionally specified in the ``unqualified-list``.  If an import which renames
-a symbol is present at module scope, imports and uses of that module will also
-be able to access that symbol using the new name instead of the old name.
-Renaming does not affect accesses to that symbol via the source module's prefix,
-nor does it affect imports or uses of that module from other contexts.  It is an
-error to attempt to rename a symbol that does not exist or is not visible in the
-respective module, or to rename a symbol to a name that is already present in
-the same ``unqualified-list``.  It is, however, perfectly acceptable to rename a
-symbol to a name present in the respective module which was not specified via
-that ``unqualified-list``.
+be given a new name using the ``as`` keyword, except for any operators.  This
+new name will be usable from the scope of the import in place of the old name
+unless the old name is additionally specified in the ``unqualified-list``.  If
+an import which renames a symbol is present at module scope, imports and uses of
+that module will also be able to access that symbol using the new name instead
+of the old name.  Renaming does not affect accesses to that symbol via the
+source module's prefix, nor does it affect imports or uses of that module from
+other contexts.  It is an error to attempt to rename a symbol that does not
+exist or is not visible in the respective module, or to rename a symbol to a
+name that is already present in the same ``unqualified-list``.  It is also an
+error to attempt to rename an operator, or to attempt to rename another symbol
+to an operator name.  It is, however, perfectly acceptable to rename a symbol to
+a name present in the respective module which was not specified via that
+``unqualified-list``.
```

Comment:
One day we could have a section about renaming that applies to both use and import (to reduce the duplication here) but I'm not insisting on it for this PR.